src/App.tsx

```diff
@@ -1,25 +1,57 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+const total = 42;
```

Comment:
The variable 'total' is hardcoded, consider retrieving this value from a state or props if it's expected to change, or from a constants file if it remains static.

Reply:
Why do I need to create separate file for 1 variable?